mockgcp/apply-proto-patches.sh

```diff
@@ -50,17 +50,3 @@ cd tools/patch-proto
 #   // The maintenance update policy determines when to allow or deny updates.
 #   MaintenanceUpdatePolicy maintenance_update_policy = 32;
 # EOF
-
-# Use our proto patch tool to add the missing pscConfig field for AlloyDB cluster.
-go run . --file ${REPO_ROOT}/mockgcp/third_party/googleapis/google/cloud/alloydb/v1beta/resources.proto --message Cluster <<EOF
```

Comment:
Oh, nice! Does this mean the AlloyDB proto now is up to date?

Reply:
Currently yes.